backend/src/main/java/com/example/cms/email/EmailSendingService.java

```diff
@@ -231,11 +232,11 @@ public void sendDeleteAccountEmail(
             MimeMessageHelper helper = new MimeMessageHelper(message, true);
             helper.setFrom(sender);
             helper.setTo(receiver.getEmail());
-            helper.setSubject("Informacja o usunięciu konta w systemie Reunice");
+            helper.setSubject("Information about deleting your account in the Eunice Platform");
```

Comment:
Myślę, że dobrym pomysłem byłoby też w przyszłości przenieść tytuły maili też do jakiegoś pliku (obojętnie, json, yml, zwykły txt) żeby można było to sobie dowolnie edytować